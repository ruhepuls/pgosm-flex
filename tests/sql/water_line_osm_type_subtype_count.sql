SELECT osm_type COLLATE "C", osm_subtype COLLATE "C", COUNT(*)
    FROM osm.water_line
    GROUP BY osm_type COLLATE "C", osm_subtype COLLATE "C"
    ORDER BY osm_type COLLATE "C", osm_subtype COLLATE "C"
;