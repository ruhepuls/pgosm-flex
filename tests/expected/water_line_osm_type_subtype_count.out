waterway|canal|14
waterway|dam|2
waterway|derelict_canal|2
waterway|ditch|5
waterway|drain|2
waterway|river|12
waterway|stream|113
waterway|weir|2
